ksql-serde/src/main/java/io/confluent/ksql/serde/Formats.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.serde;
+
+import io.confluent.ksql.serde.avro.AvroFormat;
+import io.confluent.ksql.serde.delimited.DelimitedFormat;
+import io.confluent.ksql.serde.json.JsonFormat;
+import io.confluent.ksql.serde.kafka.KafkaFormat;
+import io.confluent.ksql.util.KsqlException;
+
+/**
+ * A class containing the builtin supported formats in ksqlDB.
+ */
+public final class Formats {
```

Comment:
I take it this will grow to be a class that knows about all formats in time?

Reply:
This will know about only builtin formats - I haven't figured out exactly how we're going to do the factory (whether it be annotation/classpath based or configuring a plugin in config and packaging jars). At the point when that mechanism is worked out this class will probably be removed and we'll dogfood that mechanism with all format classes